apps/hpl/2.1/metadata.yml

```diff
@@ -46,48 +48,29 @@
     - mpi/openmpi
     - libs/atlas
 :sources:
-  - 4node_hpl.alces.dat
+  - 8proc_hpl.alces.dat
+  - 8proc_hpl_example.sh.tpl
+  - 8proc_hpl_example.sh.md
 :compile: |
   export TOPdir=`pwd`
   make arch=<%= compiler[:build_name]%> <%= redirect :make %>
 :install: |
   (cd bin && ln -snf <%=compiler[:build_name]%>/* .) <%= redirect(:install) %>
   cp -pav . <%= dest_dir %> <%= redirect(:install) %>
-  mkdir -v <%= dest_dir %>/examples <%= redirect(:install) %>
-  cp -v <%= source('4node_hpl.alces.dat') %> <%= dest_dir %>/examples <%= redirect(:install) %>
+  mkdir -pv <%= dest_dir %>/share/clusterware/templates <%= redirect(:install) %>
+  mkdir -pv <%= dest_dir %>/share/clusterware/guides <%= redirect(:install) %>
+  cp -v <%= source('8proc_hpl.alces.dat') %> <%= dest_dir %>/share/clusterware/templates <%= redirect(:install) %>
+  cp -v <%= source('8proc_hpl_example.sh.tpl') %> <%= dest_dir %>/share/clusterware/templates <%= redirect(:install) %>
+  cp -v <%= source('8proc_hpl_example.sh.md') %> <%= dest_dir %>/share/clusterware/templates <%= redirect(:install) %>
   BLAS_MAKE=`readlink <%= dest_dir %>/src/blas/Linux-gcc/Make.inc`
   rm <%= dest_dir %>/src/blas/Linux-gcc/Make.inc
   cp $BLAS_MAKE <%= dest_dir %>/src/blas/Linux-gcc/Make.inc
 :module: |
   setenv ${appcaps}DIR ${appdir}
   setenv ${appcaps}BIN ${appdir}/bin/
-  setenv ${appcaps}EXAMPLES ${appdir}/examples
+  setenv ${appcaps}EXAMPLES ${appdir}/share/clusterware/templates
+  setenv ${appcaps}EXAMPLEDOCS ${appdir}/share/clusterware/guides
 
+  prepend-path CW_DOCPATH ${appdir}/share/clusterware
   prepend-path PATH ${appdir}/bin/
   prepend-path MANPATH ${appdir}/man/
-:example_scripts:
-  - tags: ['script:type=batch']
-    name: 'hpl; tuned 4-node usage'
-    description: 'Execute HPL using an Alces-tuned input data file for 4-node/64-process operation'
-    template: 'script:stub:mpi-multinode'
-    facts:
-      hardMemoryLimit: 64G
```

Comment:
I know this was for a slightly different workload, but I think we've lost the memory limit request?  Probably want a `-l h_vmem=8G` in the example script to allow for 8G per process?